model-optimizer/extensions/front/tf/RetinaNetFilteredDetectionsReplacement.py

```diff
@@ -246,9 +246,17 @@ def generate_sub_graph(self, graph: Graph, match: SubgraphMatch):
                                                                    applied_width_height_regressions_node)
 
         detection_output_op = DetectionOutput(graph, match.custom_replacement_desc.custom_attributes)
+        # get nms from original network
+        for node_name in match.matched_nodes_names():
+            if Node(graph, node_name)['op'] == 'NonMaxSuppression':
+                iou_threshold = Node(graph, node_name).in_node(3).value
+                break
+        if not iou_threshold:
```

Comment:
if `iou_threshold` is 0.0 then this condition is True. Please, change to `if iou_threshold is not None:`

Reply:
Agree.